From e27e7fc95bf89adbf91b17389c2c9ef56bf59d85 Mon Sep 17 00:00:00 2001
From: Marcin Niestroj <m.niestroj@emb.dev>
Date: Fri, 23 Feb 2024 14:35:48 +0100
Subject: [PATCH] samples: net: echo_server: add native_sim to allowed
 platforms

'native_sim' builds without any issues as part of twister and it also works
when manually setting up Ethernet virtual device (using
tools/net-tools/net-setup.sh).

Signed-off-by: Marcin Niestroj <m.niestroj@emb.dev>
---
 samples/net/sockets/echo_server/sample.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/samples/net/sockets/echo_server/sample.yaml b/samples/net/sockets/echo_server/sample.yaml
index 1174797d04a..c2ff08fa204 100644
--- a/samples/net/sockets/echo_server/sample.yaml
+++ b/samples/net/sockets/echo_server/sample.yaml
@@ -18,6 +18,7 @@ tests:
       - qemu_cortex_m3
       - frdm_kw41z
       - qemu_cortex_a53
+      - native_sim
     integration_platforms:
       - qemu_x86
   sample.net.sockets.echo_server.802154:
-- 
2.43.2

